src/doc/conf.py

```diff
@@ -68,7 +68,8 @@ def setup(app):
 # Best place to put a substitution for 'VisIt'
 # Authors should use either VisIt_ or VisIt_'s
 rst_epilog = """
-.. _VisIt: https://visit-dav.github.io/visit-website/
+.. _VisIt: https://visit.llnl.gov
```

Comment:
Why the change to VisIt's website?

Reply:
Did you read the initial comment 😉? I feel our *alias* is a better one to use than the *specific* github URL. Its more enduring.

Reply:
Apologies, you are right, I hadn't read the comment.